py2many/scope.py

```diff
@@ -72,6 +72,9 @@ def find_import(self, lookup):
                     if imp.name == lookup:
                         return imp
 
+    def parent(self):
```

Comment:
IMO this should be `parent_scopes` as `parent` sounds like it returns a single scope, and could be a `@property`.